graspologic/align/seeded_procrustes.py

```diff
@@ -0,0 +1,186 @@
+import time
+
+import numpy as np
+
+from graspologic.align import OrthogonalProcrustes, SeedlessProcrustes
+
+from .base import BaseAlign
+
+
+class SeededProcrustes(BaseAlign):
+
+    """
+    Matches two datasets using an orthogonal matrix and a '2d' matrix of seeds. Unlike
+    :class:`~graspologic.align.OrthogonalProcrustes` or :class:`~graspologic.align.SeedlessProcrustes`,
+    this requires only a partial matching between entries. It can even be used in
+    the settings where the two datasets do not have the same number of entries.
+
+    In the graph setting, it is used to align the embeddings of two different
+    graphs, which requires some simultaneous inference task and partial 1-1
+    matching between the vertices of the two graphs.
```

Comment:
I would say something like 

"Aligns two datasets when a partial matching of entries between the two datasets is known." and that's it

Reply:
i should add that sentence instead of lines 13-20?

Reply:
yes